deploy/nri-prometheus.tmpl.yaml

```diff
@@ -83,40 +83,55 @@ data:
     # The name of your cluster. It's important to match other New Relic products to relate the data.
     cluster_name: "<YOUR_CLUSTER_NAME>"
 
-    # When standalone is set to false nri-prometheus requires an infrastructure agent to work and send data. Defaults to true
+    # When standalone is set to false nri-prometheus requires an infrastructure agent to work and send data. 
+    # Default: true
     # standalone: true
 
-    # How often the integration should run. Defaults to 30s.
+    # How often the integration should run. 
+    # Default: "30s"
     # scrape_duration: "30s"
 
-    # The HTTP client timeout when fetching data from endpoints. Defaults to 5s.
+    # The HTTP client timeout when fetching data from endpoints. 
+    # Default: "5s"
     # scrape_timeout: "5s"
 
     # How old must the entries used for calculating the counters delta be
-    # before the telemetry emitter expires them. Defaults to 5m.
+    # before the telemetry emitter expires them. 
+    # Default: "5m"
     # telemetry_emitter_delta_expiration_age: "5m"
 
     # How often must the telemetry emitter check for expired delta entries.
-    # Defaults to 5m.
+    # Default: "5m"
     # telemetry_emitter_delta_expiration_check_interval: "5m"
 
-    # Wether the integration should run in verbose mode or not. Defaults to false.
+    # Wether the integration should run in verbose mode or not. 
+    # Default: false
     verbose: false
 
     # Whether the integration should run in audit mode or not. Defaults to false.
     # Audit mode logs the uncompressed data sent to New Relic. Use this to log all data sent.
     # It does not include verbose mode. This can lead to a high log volume, use with care.
+    # Default: false
     audit: false
 
-    # Wether the integration should skip TLS verification or not. Defaults to false.
+    # Wether the integration should skip TLS verification or not. 
+    # Default: false
     insecure_skip_verify: false
 
-    # The label used to identify scrapable targets. Defaults to "prometheus.io/scrape".
+    # The label used to identify scrapable targets.
+    # Targets can be identified using a label or annotation.
+    # Default: "prometheus.io/scrape"
     scrape_enabled_label: "prometheus.io/scrape"
 
-    # Whether k8s nodes need to be labelled to be scraped or not. Defaults to true.
+    # Whether k8s nodes need to be labelled to be scraped or not. 
+    # Default: true
     require_scrape_enabled_label_for_nodes: true
 
+    # Number of worker threads used for scraping targets.
+    # For large clusters with many endpoints, increase to 10.
```

Comment:
@bpschmitt we are currently working on a load test and posible refactor to improve performance related to the number of workers. We have detected that increasing this number without reducing the `emitter_harvest_period` could lead to a high increase of memory consumption. Also the core consumption peak is something that needs to be taken into account. We plan to write a general recommendation for this configuration.